ginga/misc/plugins/Cuts.py

```diff
@@ -120,6 +122,7 @@ def __init__(self, fv, fitsimage):
                                   show_cuts_legend=False, enable_slit=False)
         self.settings.load(onError='silent')
         self.colors = self.settings.get('colors', cut_colors)
+        self.show_legend = self.settings.get('show_cuts_legend', False)
```

Comment:
Or perhaps I can move `self.show_legend = self.settings.get('show_cuts_legend', False)` from `__init__` to `build_gui()`?